components/containers/DynamicForm/DynamicForm.tsx

```diff
@@ -41,6 +38,11 @@ export const DynamicForm = (props: DynamicFormProps): React.ReactElement => {
       <Head>
         <title>{formTitle}</title>
       </Head>
+      {isReCaptchaEnableOnSite ? (
```

Comment:
Cleaner to use: 

`{isReCaptchaEnableOnSite && <Script src={reChaptchaScriptURL} strategy="beforeInteractive" /> }`